Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -1022,3 +1018,14 @@ noncomputable def realLinearIsometryEquiv (h : I = (0 : K)) : K ≃ₗᵢ[ℝ] 
 end CaseSpecific
 
 end RCLike
+
+namespace AddChar
+variable {G : Type*} [Finite G]
+
+lemma inv_apply_eq_conj [AddLeftCancelMonoid G] (ψ : AddChar G K) (x : G) : (ψ x)⁻¹ = conj (ψ x) :=
+  RCLike.inv_eq_conj <| norm_apply _ _
+
+lemma map_neg_eq_conj [AddCommGroup G] (ψ : AddChar G K) (x : G) : ψ (-x) = conj (ψ x) := by
+  rw [map_neg_eq_inv, inv_apply_eq_conj]
```

Comment:
Why is it `inv_apply` and `map_neg` (and not `apply_neg`)?

Reply:
Lemmas of the form `free_variable (op x ... z) = ...` are called `map_op`. Lemmas of the form `op (f x) ... (f z) = ...` are called `op_apply`.

Reply:
I believe you, but I don't think this is very consistent...